code/game/atoms.dm

```diff
@@ -563,6 +563,25 @@
 					for(var/datum/reagent/R in reagents.reagent_list)
 						total_volume += R.volume
 					. += "[total_volume] units of various reagents"
+				//-------- Beer goggles ---------
+				if(user.can_see_boozepower())
+					var/total_boozepower = 0
+					var/list/taste_list = list()
+					if(length(reagents.reagent_list))
```

Comment:
`list()` is falsy afaik, so just check the list, no need for length

Reply:
It's `reagents.reagent_list` to check, not `taste_list` to check

Reply:
ah, nevermind. you're right.

Reply:
fixed